### DISCUSSION TITLE
keyboard shortcut

### DISCUSSION THREAD

**Discussion by rad182** [vitest-dev/vscode#344] [Q&A] [answered]
Hello, is there a keyboard shortcut to run like nearest test if the cursor is within the test?

**Comment by sheremet-va** [comment, marked as answer]
There is "run test at cursor" shortcut. For me it's like this:

![Screenshot 2024-04-15 at 16 37 55](https://github.com/vitest-dev/vscode/assets/16173870/5613c4ad-3538-4362-8b40-0a86594c34ad)

**Reply by rad182** [reply]
awesome! didn't know about this. Thank you very much!
